### PR TITLE
Fix test case in unit test result import.

### PR DESCRIPTION
**Pull request by lindamartin**

### CODE REVIEW

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java [hunk 1]
```diff
@@ -114,24 +122,22 @@ public void testGetUnitTestFileName() {
...
     InputFile inputFile1 = sensor.getTestFileRelativePathToBaseDir("PersonTest.js");
     assertNotNull(inputFile1);
-    assertEquals("PersonTest.js", inputFile1.relativePath());
+    assertEquals("test/PersonTest.js", inputFile1.relativePath());
   }
 
   @Test
   public void get_testfile_with_directory() {
-    initFilesystem();
+    System.out.println("TEST_DIR");
```
Comment (vilchik-elena) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @lindada do we need this sout?

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java [hunk 2]
```diff
@@ -139,18 +145,12 @@ public void test_toString() {
     assertThat(sensor.toString()).isEqualTo("JsTestDriverSensor");
   }
 
-  public DefaultInputFile newTestInputFile(String name, String path) {
-    return new DefaultInputFile(name)
+  public DefaultInputFile newTestInputFile(String relativePath, String path) {
```
Comment (vilchik-elena) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @lindamartin It's not clear what is the difference between *relativePath* and *path*

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java (hunk 1 above)
Comment (lindamartin) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @vilchik-elena no indeed, fixed in 2df39e8bb841afaf1ed732da84311700268f415c.

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java (hunk 2 above)
Comment (lindamartin) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @vilchik-elena tried to renamed it in 2df39e8bb841afaf1ed732da84311700268f415c. Is it a little better ?

Comment (vilchik-elena) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @lindamartin yes, much better!

File: pom.xml [hunk 3]
```diff
@@ -65,8 +65,6 @@
   <properties>
     <license.title>SonarQube JavaScript Plugin</license.title>
 
-    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```
Comment (vilchik-elena) [inline pom.xml]: @lindamartin why did you remove this line?

Comment (pynicolas) [inline pom.xml]: @vilchik-elena That may be useful to understand why a test fails in Travis

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java [hunk 4]
```diff
@@ -114,24 +122,20 @@ public void testGetUnitTestFileName() {
...
 
   @Test
   public void get_testfile_with_directory() {
-    initFilesystem();
-
-    InputFile inputFile1 = sensor.getTestFileRelativePathToBaseDir("AnotherPersonTest.js");
+    InputFile inputFile1 = sensor.getTestFileRelativePathToBaseDir("PersonTest.js");
     assertNotNull(inputFile1);
-    assertEquals("AnotherPersonTest.js", inputFile1.relativePath());
+    assertEquals("test/PersonTest.js", inputFile1.relativePath());
```
Comment (vilchik-elena) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @lindamartin you are testing the same thing here and on the lines 125-127

**Comment by vilchik-elena** [comment]
LGTM!

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java (hunk 4 above)
Comment (lindamartin) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/unittest/jstestdriver/JsTestDriverSensorTest.java]: @vilchik-elena indeed good catch thanks! Fixed in 9bb96e99a6c67ed2794a2d67cf2b6cb6a6b0e304

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/java/org/sonar/plugins/javascript/api/visitors' package (org.sonarsource.javascript:javascript-squid:src/main/java/org/sonar/plugins/javascript/api/visitors) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by lindamartin** [comment]
Merged in 017458467decd317cfb9323973868617aed82005.
